app/src/main/java/com/github/h3lp3rs/h3lp/forum/ForumCategory.kt

```diff
@@ -23,17 +29,31 @@ enum class ForumCategory {
         fun forumOf(choice: ForumCategory): Forum {

```

Comment:
forumOf could be changed into a simple call to root().child(choice.name) (this would avoid the code duplication too)